package.json

```diff
@@ -13,6 +13,7 @@
     "test:e2e:demo": "cd ./e2e-tests && npm run test:e2e:demo",
     "test:e2e:postprocess": "cd ./e2e-tests && npm run test:e2e:postprocess",
     "test:e2e:files": "cd ./e2e-tests && ./create-large-test-files.sh",
+    "test:e2e:smoke": "cd ./e2e-tests && BASEURL=$BASEURL npm run test:e2e:smoke",
```

Comment:
Is `BASEURL=$BASEURL` necessary to get it to run on Windows? Shouldn't be necessary on Mac and isn't needed on Linux

Reply:
that's the variable i'm proposing we use to point this at different environments.
the "OOB" behaviour of running the e2e tests is that it has the baseurl set to localhost:9000; this is exposing some variable that we can use to point everything at 'something else'.

the assumption is that you'll have a step that does a deploy, then a step that does a `BASEURL={url we just deployed to} npm run blah`

Reply:
sorry, realised the confusion.
this is with the assumption that you're going to pass that value in when you call things, so in root:
`BASEURL=https://appeals-dev.planninginspectorate.gov.uk/ npm run test:e2e:smoke`
and the entry here is just passing that config up the chain